Content.Shared/Humanoid/NamingSystem.cs

```diff
@@ -47,20 +47,20 @@ public string GetFirstName(SpeciesPrototype speciesProto, Gender? gender = null)
             switch (gender)
             {
                 case Gender.Male:
-                    return _random.Pick(_prototypeManager.Index<DatasetPrototype>(speciesProto.MaleFirstNames).Values);
+                    return Loc.GetString(_random.Pick(_prototypeManager.Index<LocalizedDatasetPrototype>(speciesProto.MaleFirstNames).Values));
```

Comment:
```suggestion
                    return _random.Pick(_prototypeManager.Index<LocalizedDatasetPrototype>(speciesProto.MaleFirstNames));
```
You can simplify these a little - there's a helper extension that'll do the GetString call.

You'll probably need to add `using Content.Shared.Random.Helpers`